daemon/list.go

```diff
@@ -369,8 +369,9 @@ func (daemon *Daemon) foldFilter(ctx context.Context, view *container.View, conf
 		isTask:         isTask,
 		publish:        publishFilter,
 		expose:         exposeFilter,
-		ListOptions:    config,
 		names:          view.GetAllNames(),
+
+		ContainerListOptions: config,
```

Comment:
Maybe we should un-embed this one, but haven't looked closely yet if that would require a lot of changes, so keeping that for a future exercise